src/net/sourceforge/kolmafia/textui/Parser.java

```diff
@@ -4305,6 +4326,54 @@ public static String getLineAndFile(final String fileName, final int lineNumber)
     return "(" + fileName + ", line " + lineNumber + ")";
   }
 
+  public static final String getFileAndRange(String fileName, final Range range) {
+    if (range == null || Positions.isBefore(range.getEnd(), range.getStart())) {
+      throw new IllegalArgumentException();
+    }
+
+    final StringBuilder result = new StringBuilder();
+
+    result.append("(");
```

Comment:
Tiny nitpick but don't add the parens in this function.

Reply:
This is just conforming to the behaviour of 
```java
  public static String getLineAndFile(final String fileName, final int lineNumber) {
    if (fileName == null) {
      return "(" + Preferences.getString("commandLineNamespace") + ")";
    }

    return "(" + fileName + ", line " + lineNumber + ")";
  }
```
(which is just above this one)

Reply:
I think that's bad behaviour IMO